README.md

```diff
@@ -69,7 +69,7 @@ Report bugs or submit feature requests in GitHub [Issues].  Please use one of th
 
 ## Contributing
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+This project welcomes contributions and suggestions and guidance can be found in our [contribution guidelines]. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
```

Comment:
This project welcomes contributions and suggestions.  Guidance can be found in our [contribution guidelines].

Reply:
This?
```suggestion
This project welcomes contributions and suggestions. Guidance can be found in our [contribution guidelines]. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
```